transports/sqs/index_transport-at-a-glance_sqs_[5,).partial.md

```diff
@@ -0,0 +1,11 @@
+## Transport at a glance
+
+|Feature                    |   |  
+|:---                       |---
+|Transactions |None, ReceiveOnly (Message visibility timeout)
+|Pub/Sub                    |Native (Requires SNS, supports hybrid-mode for migration purposes)
+|Timeouts                   |Native (Requires FIFO Queues)
+|Large message bodies       |Native (Requires S3)
+|Scale-out             |Competing consumer
+|Scripted Deployment        |PowerShell, Terraform, C#
```

Comment:
```
|Scripted Deployment        | Tool, CloudFormation, C#
```

Aren't we planning to drop powershell support anyway?

Reply:
Yes, I haven't started the CLI documentation, yet